Tiltfile

```diff
@@ -24,11 +18,17 @@ settings.update(read_json(
     default = {},
 ))
 
+allow_k8s_contexts(settings.get("allowed_contexts"))
+
+envsubst_cmd = "./hack/tools/bin/envsubst"
+kustomize_cmd = "./hack/tools/bin/kustomize"
+yq_cmd = "./hack/tools/bin/yq"
+os_name = local_output("go env GOOS")
```

Comment:
Just to confirm. So local_output fails if we don't call allow_k8s_contexts before?

Can you post the error? I"m a bit confused as I never did get the error.

Reply:
The error is following:
```
Loading Tiltfile at: /cluster-api/Tiltfile
ERROR: Traceback (most recent call last):
  /cluster-api/Tiltfile:10:23: in <toplevel>
  /cluster-api/tilt_modules/local_output/Tiltfile:2:19: in local_output
  <builtin>: in local
Error: Refusing to run 'local' because test-context might be a production kube context.
If you're sure you want to continue add:
        allow_k8s_contexts('test-context')
before this function call in your Tiltfile. Otherwise, switch k8s contexts and restart Tilt.
```

Even though my `tilt-settings.json` includes `"allowed_contexts": ["test-context"],`. With this commit the issue is fixed.

Reply:
That is very interesting, sounds like `tilt up` shouldn't work anywhere right now?

Reply:
Hmm, I wonder, maybe with kind cluster somehow it works just fine and this is what people usually use?

I use the following Tilt version:
```
$ tilt version
v0.22.15, built 2021-10-29
```

Reply:
Maybe. I'm using it with kind and tilt `v0.22.14`

Reply:
P.S. Change seems fine to me, I'm mostly curious under which circumstances the problem occurs.

Reply:
Yep, just checked and `tilt up` works just fine with kind cluster.

Reply:
I suppose it's something around this code: https://github.com/tilt-dev/tilt/blob/a9eba86c2dd120b2338d5475db85c4ee9b38fbba/internal/k8s/env.go#L108-L110.

Reply:
Thx, that makes sense!